template-processors/ocp-template/Dockerfile.in

```diff
@@ -1,10 +1,10 @@
 FROM @REPOSITORY@/eunomia-base:@IMAGE_TAG@
 
-ENV OC_VERSION=3.11.170
+ENV OC_VERSION=4.4
```

Comment:
Why 4.4?

Reply:
Why not 4.6? Why not Latest 😨 ?

Reply:
My thought was to update everything to k8s 1.17. k8s 1.17 is the version currently available in the GKE regular release channel. My thought was to update all of the clients in the template processor images to be compatible with the same k8s version. Therefore OCP 4.4 which is k8s 1.17.

Reply:
Updated to kubectl 1.19 and oc 4.6.